salt/_modules/metalk8s.py

```diff
@@ -78,6 +78,13 @@ def add_etcd_node(host, endpoint):
         host (str): hostname of the new etcd node
         endpoint (str): endpoint of the new etcd node
     '''
+    if not endpoint:
+        # If we have no endpoint get it from mine
+        node_ip = __salt__['saltutil.runner'](
```

Comment:
Can we use a `runner` in a `module`? :confused:

Reply:
Yes, that works

Reply:
It's maybe not the best practice but we will likely remove this in #877 so for me it's ok